src/test/rustdoc/deref-const-fn.rs

```diff
@@ -0,0 +1,23 @@
+#![crate_name = "foo"]
+
+// @has 'foo/struct.Bar.html'
+pub struct Bar;
+
+impl Bar {
+    // @has - '//*[@id="method.len"]' 'pub const fn len(&self) -> usize'
+    pub const fn len(&self) -> usize { 0 }
+}
+
+pub struct Foo {
+    value: Bar,
+}
+
+impl std::ops::Deref for Foo {
+    type Target = Bar;
+
+    // @has 'foo/struct.Foo.html'
+    // @has - '//*[@id="method.len"]' 'pub fn len(&self) -> usize'
```

Comment:
nit: Maybe move this to just before the `impl` since it's not directly connected to the `deref()` function?

Reply:
:+1: